cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -3028,12 +3028,77 @@
 
   <property>
     <name>app.program.runtime.monitor.topics.configs</name>
-    <value>audit.topic,data.event.topic,metadata.messaging.topic,metrics.topic.prefix:${metrics.messaging.topic.num},program.status.event.topic,program.status.event.topic:${program.status.event.topic.num.partitions},log.tms.topic.prefix:${log.publish.num.partitions}</value>
+    <value>app.delete.event.topic,audit.topic,data.event.topic,metadata.messaging.topic,metrics.topic.prefix:${metrics.messaging.topic.num},program.status.event.topic,program.status.event.topic:${program.status.event.topic.num.partitions},log.tms.topic.prefix:${log.publish.num.partitions}</value>
     <description>
       A comma-separated list of topic config to be monitored by runtime monitor
     </description>
   </property>
 
+  <property>
+    <name>app.delete.event.topic</name>
+    <value>appdeleteevent</value>
```

Comment:
this is hard to read, can we rename it to appdeletes

Reply:
do you mean `appdeleteevent` -> `appdeletes`? Should it be `appdelete` instead?

Reply:
either one is fine, it's just visually difficult with the 2 e's in a row.